ResidentInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -117,16 +150,25 @@ private static AddressResponse ToResponse(this Address domain)
             };
         }
 
-        private static List<EmailResponse> ToResponse(this List<Email> emails)
+        private static AddressResponse ToResponse(this HousingAddress domain)
+        {
+            return new AddressResponse
+            {
+                AddressLine1 = domain.AddressLine1,
```

Comment:
We're missing `PropertyRef` here. :)

Reply:
It wasn't in the response currently for this API so I assumed it might not be needed as all the address objects are the same. I'm not sure about this though.

Reply:
I see. Yeah, that makes sense. We could have a separate PR for updating gateway tests if we need too.